### ISSUE TITLE
Mis-hit reported on class declaration due to explicit destructor present for one of members.

### ISSUE THREAD

**Issue by AmokHuginnsson** [linux-test-project/lcov#50]
Consider the following minimal example:

(sample.cxx)
```C++
struct V {
        ~V() {}
};

struct A {
        V _v;
};

int main( int argc_, char** ) {
        A a;
        return 0;
}
```

Compile with `g++ --coverage sample.cxx`, generate coverage report with:
```Console
lcov -b . -d . -c -i -o base.info
./a.out
lcov -b . -d . -c -o test.info
lcov -a base.info -a test.info -o coverage.info
genhtml --legend coverage.info -o html
```
This results in following report:
```
          Line data    Source code

       1             : struct V {
       2           1 :         ~V() {}
       3             : };
       4             : 
       5           0 : struct A {
       6             :         V _v;
       7             : };
       8             : 
       9           1 : int main( int argc_, char** ) {
      10           1 :         A a;
      11           1 :         return 0;
      12             : }
      13             : 
```

Observe `0` hits reported for `struct A` declaration at line `5`.
This problem only manifests on GCC/gcov 8, previous version of GCC/gcov reports no active code at line `5`.

**Comment by oberpar** [comment]
Thanks for the report. This should be fixed with commit 9aa0d14af4446ef46d80356849a97bc961a91f97.

**Comment by AmokHuginnsson** [comment]
Yes! This fixes the issue.
Thank you!
